readthedocs/invitations/templates/invitations/email/invitation.txt

```diff
@@ -0,0 +1,10 @@
+{% extends 'core/email/common.txt' %}
+
+{% block content %}
+{{ from_name }} has invited you to join the {{ object_description }}.
```

Comment:
```suggestion
{{ from_name }} has invited you to join {{ object_description }}.
```

Reply:
Also same note here, removing the article sounds incorrect, as `object_description` includes a noun too -- it's not just "Read the Docs", but "Read the Docs organization".